site-src/guides/getting-started.md

```diff
@@ -25,34 +25,38 @@ network infrastructure that your Gateway resources represent). Note that many
 of the Gateway controller setups will install and remove the Gateway API bundle
 for you.
 
-## Installing a Gateway API Bundle
+## Installing Gateway API
 
 A Gateway API bundle represents the set of CRDs and validating webhook
-associated with a version of Gateway API.
+associated with a version of Gateway API. Each release includes two
+channels with different levels of stability:
 
-### Install the CRDs
+### Install Standard Channel
 
-The following command will install the Gateway API CRDs. This includes
-GatewayClass, Gateway, HTTPRoute, TCPRoute, and more. Note that a running
-Gateway controller in your Kubernetes cluster is required to actually act on
-these resources. Installing the CRDs will just allow you to see and apply the
-resources, but they won't do anything without a controller implementing them.
+The standard release channel includes all resources that have graduated to beta,
+including GatewayClass, Gateway, and HTTPRoute. To install this channel, run the
+following kubectl command:
```

Comment:
```suggestion
The standard release channel includes all resources that have at least graduated
to beta, including GatewayClass, Gateway, and HTTPRoute. To install this
channel, run the following kubectl command:
```

Reply:
I think "at least beta" could confuse users in suggesting that something in our API has graduated beyond beta, maybe we can save this update for when that happens?

Reply:
ok :+1: